use_cases/eluc/data/constants.py

```diff
@@ -0,0 +1,85 @@
+from pathlib import Path
+
+import pandas as pd
+import regionmask
+
+# This should work if python path is set to ELUC
+ROOT_DIR = Path(".")
+DATA_FILE_PATH = ROOT_DIR / "data/merged_aggregated_dataset_1850_2022.zarr.zip"
+UPDATE_FILE_PATH = ROOT_DIR / "data/BLUE_LUH2-GCB2022_ELUC-committed_gridded_net_1850-2021.nc"
+CODES_PATH = ROOT_DIR / "data/codes.csv"
+
+LAND_USE_COLS = ['c3ann', 'c3nfx', 'c3per','c4ann', 'c4per', 'pastr', 'primf', 'primn', 'range', 'secdf', 'secdn', 'urban']
+
+# TODO: Temporary to merge the crops together
+CROP_COLS = ['c3ann', 'c3nfx', 'c3per','c4ann', 'c4per']
+LAND_USE_COLS = ["crop"] + [col for col in LAND_USE_COLS if col not in CROP_COLS]
+
+DIFF_LAND_USE_COLS = [f"{col}_diff" for col in LAND_USE_COLS]
+COLS_MAP = dict(zip(LAND_USE_COLS, DIFF_LAND_USE_COLS))
+
+# RECO_COLS = ['c3ann', 'c3nfx', 'c3per','c4ann', 'pastr', 'range', 'secdf', 'secdn']
+NO_CHANGE_COLS = ["c4per_diff", "primf_diff", "primn_diff", "urban_diff"]
+RECO_COLS = [col.split("_")[0] for col in DIFF_LAND_USE_COLS if col not in NO_CHANGE_COLS]
+DIFF_RECO_COLS = [f"{col}_diff" for col in RECO_COLS]
+RECO_MAP = dict(zip(RECO_COLS, DIFF_RECO_COLS))
+
+NONLAND_FEATURES = ["cell_area", "lat", "lon", "time"]
+
+# TODO: Updated this to include lat lon time
+# NN_FEATS = LAND_USE_COLS + ["cell_area"] + DIFF_LAND_USE_COLS
+NN_FEATS = LAND_USE_COLS + NONLAND_FEATURES + DIFF_LAND_USE_COLS
+
+EU_COUNTRIES = ["GB", "FR", "DE", "NL", "BE", "CH", "IE"]
```

Comment:
It would be nice to have a comment above EU_COUNTRIES that gives the full name of the countries, like for south american countries below